### PR TITLE
Remove NotSupportedException class

### PR DESCRIPTION
**Pull request by vbauer**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@vbauer__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

File: src/main/java/microsoft/exchange/webservices/data/security/SafeXmlDocument.java [hunk 1]
```diff
@@ -67,12 +66,11 @@ public SafeXmlDocument() {
...
    * specified XSImplementation.
    *
-   * @param imp The XmlImplementation to use.
-   * @throws NotSupportedException
+   * @param imp The XmlImplementation to use
    */
   // <remarks>Not supported do to no use within exchange dev code.</remarks>
-  public SafeXmlDocument(DocumentBuilder imp) throws NotSupportedException {
-    throw new NotSupportedException("Not supported");
+  public SafeXmlDocument(final DocumentBuilder imp) {
```
Comment (serious6) [inline src/main/java/microsoft/exchange/webservices/data/security/SafeXmlDocument.java]: why do we have this constructor? Seems useless to me.

Comment (vbauer) [inline src/main/java/microsoft/exchange/webservices/data/security/SafeXmlDocument.java]: I have no idea. :) +1 to remove it.

Comment (serious6) [inline src/main/java/microsoft/exchange/webservices/data/security/SafeXmlDocument.java]: :ship:

**Comment by vbauer** [comment]
Updated.

**Comment by vboctor** [comment]
:+1:

**Comment by serious6** [comment]
@vbauer thanks for the contribution

**Pull request merged**